site/docs/uk/plugins/emoji.md

```diff
@@ -0,0 +1,108 @@
+# Емодзі (`emoji`)
+
+За допомогою цього плагіна ви можете легко вставляти емодзі у свої відповіді, шукаючи їх безпосередньо у редакторі коду, замість того, щоб вручну копіювати і вставляти емодзі з Інтернету у свій код.
+
+## Чому я маю його використовувати?
+
+Чому б ні? Люди постійно використовують емодзі у своєму коді, щоб проілюструвати повідомлення, яке вони хочуть надіслати, або щоб упорядкувати дещо.
+Але ви щоразу втрачаєте фокус, коли вам потрібен новий емодзі:
+
+1. Ви припиняєте написання коду, щоб розпочати пошук конкретного емодзі.
+2. Ви заходите в чат Telegram і витрачаєте близько 6-ти секунд, не більше, на пошук потрібного вам емодзі.
+3. Ви копіюєте його у свій код і повертаєтесь до написання коду, але з втраченим фокусом.
+
+З цим плагіном ви не тільки не припиняєте писати код, але й не втрачаєте фокус.
+Існують також системи та/або редактори, які не люблять й не показують емодзі, тому в результаті ви вставляєте білий квадрат на кшталт цього: `Я такий щасливий □`.
+
+Цей плагін має на меті вирішити ці проблеми, виконуючи за вас складне завдання парсингу емодзі в усіх системах і дозволяючи вам лише шукати їх за допомогою доступного зручного автодоповнення. Тепер вищеописані кроки можна звести до цього:
```

Comment:
The original here says:

_This plugin aim to solve these issues, handling for you the hard task of parsing emojis in all systems and letting you only search for them in a easy way (autocomplete is available)._

What we mean by parsing is that source files might be stored different encodings on different operating systems, and they are interpreted differently with different editors. We also mean that you often need to copy them from the web to be sure that you did not screw up. I think the word parsing is not such a great pick in English, but I also can't think of anything better. Encoding maybe, but that's not very accurate, either.

The part about searching is that you sometimes don't know the exact alpha names of the emoji you want to use. The plugin offers autocomplete so if you know that you want a thinking emoji, just start typing _think_ and it will tell you that the alpha name is `thinking_face`.

Reply:
@Nazar-Ant, we can leave it that way, we'll find a better description of the action in the future.

Reply:
And only now I see the typo in the English docs 🫠